### ISSUE TITLE
Symfony 5 deprecation

### ISSUE THREAD

**Issue by crabnky** [msgphp/msgphp#392]
Recently I updated my project to Symfony 5, and got this:
```
The "Symfony\Bundle\SecurityBundle\DataCollector\SecurityDataCollector" class is considered final. It may change without further notice as of its next major version. You should not extend it from "MsgPhp\User\Infrastructure\Security\DataCollector".
```
:)

**Comment by ro0NL** [comment]
HI @crabnky see this duplicate issue https://github.com/msgphp/msgphp/issues/371, fixed in master current, and will be released in v0.16 soonish :).

The fix in bfb1b4c38af5e46caa66b5d325d70e9565ffe6e0 simply removes the feature for now btw.

**Comment by crabnky** [comment]
OK, my bad, did not check other issues :) Sorry for duplicate.
